monitoring/checkers.go

```diff
@@ -59,6 +59,12 @@ func NodesStatusHealth(config KubeConfig, nodesReadyThreshold int) health.Checke
 	return NewNodesStatusChecker(config, nodesReadyThreshold)
 }
 
+// PingHealth creates a checker that monitors ping values between Master nodes
+// and other nodes
+func PingHealth(serfRPCAddr string, serfRPCName string) health.Checker {
```

Comment:
Nitpick: You could've used shorter form for function arguments:
```go
func PingHealth(serfRPCAddr, serfRPCName string) health.Checker
```

Reply:
nice!